src/microbe_stage/editor/HexPopupMenu.cs

```diff
@@ -119,6 +120,7 @@ public override void _Ready()
 
     public override void _EnterTree()
     {
+        normalFont = GetFont("normal", "Fonts");
```

Comment:
This should get the font from the titleLabel in CalculateSize and only use this normal font if there's no font set on the titleLabel.